core/unix/signal_linux_x86.c

```diff
@@ -595,6 +597,18 @@ mcontext_to_sigcontext_simd(sig_full_cxt_t *sc_full, priv_mcontext_t *mc)
                            YMMH_REG_SIZE);
                 }
             }
+            /* Lazy signal state can stop us from setting the full state.
+             * Switch the signal to full state if necessary.
+             */
+            uint bv_high, bv_low;
```

Comment:
Is this meant to be a temporary solution until we figure out why subsequent signals in static DR don't have the "correct" xstate_bv value?

Some more context in this comment would help future readers.

Reply:
No, the assumption is there is some real kernel local laziness and we need this fix permanently.  Added more comments.